runtime/jvmti/jvmtiThread.c

```diff
@@ -219,13 +219,17 @@ jvmtiSuspendThread(jvmtiEnv* env,
 
 	rc = getCurrentVMThread(vm, &currentThread);
 	if (rc == JVMTI_ERROR_NONE) {
-		UDATA currentThreadSuspended;
+		BOOLEAN currentThreadSuspended;
```

Comment:
A local variable should be initialized in its declaration. See also lines 264 & 277 (and others which trust you'll address in #15914).

Reply:
All formatting comments are addressed in https://github.com/eclipse-openj9/openj9/pull/15945.